app/views/support_interface/duplicate_candidate_matches/index.html.erb

```diff
@@ -0,0 +1,3 @@
+<%= render 'support_interface/candidates/candidates_navigation', title: 'Duplicate candidate matches' %>
+
+<%= render SupportInterface::DuplicateCandidateMatchesTableComponent.new(matches: @matches) %>
```

Comment:
Should we add a bit of text saying something like "Applications appear in this list when they have postcode, name and date of birth in common, but they were created using different candidate accounts"